.github/workflows/test.yml

```
@@ -49,10 +49,78 @@ jobs:
           DANGER_GITHUB_API_TOKEN: ${{ secrets.DANGER_GITHUB_API_TOKEN }}
         run: bundle exec danger
 
+  ios_test:
+    name: Running iOS tests
+    runs-on: macos-latest
+    defaults:
+      run:
+        working-directory: ./sample/ios
+    steps:
+      - name: Check out
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+
+      - name: Set up cache
+        uses: actions/cache@v3
+        id: bunlderCache
+        with:
+          path: vendor/bundle
+          key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-gems-
+
+      - name: Setup Java JDK
+        uses: actions/setup-java@v2.1.0
+        with:
+          distribution: 'adopt'
+          java-version: '17'
+
+      - name: Remove the old sample project
+        run: rm -rf sample
+        working-directory: ./
+
+      - name: Generate the new sample project
+        run: sh make.sh --bundle-id co.nimble.kmm.sample --bundle-id-staging co.nimble.kmm.sample.staging --project-name sample --ios-version 14.0
+        working-directory: ./
+
+      - name: Bundle install
+        run: bundle install --path vendor/bundle
```

The default path is `vendor/bundle` https://bundler.io/v1.12/man/bundle-config.1.html#LIST-OF-AVAILABLE-KEYS

```suggestion
        run: bundle install
```